README.md

```diff
@@ -83,52 +85,152 @@ When running in Visual Studio Code, Sinatra tends to run on port 9292.
 
 The settings for Visual Studio Code can be found in the `launch.json` file.
 
-### Running the docker container
+#### Running w/ Docker
```

Comment:
Ahh good point. I do forget to add that. I may add the shields with pull and star counts as well.